openwisp_monitoring/monitoring/tasks.py

```diff
@@ -36,6 +36,11 @@ def timeseries_write(
         post_metric_write.send(**signal_kwargs)
 
 
+@shared_task
+def delete_device_tsdb_data(key, tags):
```

Comment:
Why are you splitting some code in `openwisp_monitoring.device` and some other lines in `openwisp_monitoring.monitoring`?

It could make sense to have a generic way to remove timeseries data asynchronously, I like the idea, but this task shouldn't have `device` in its name, I'd rename it simply `timeseries_delete`, which is also more consistent with the naming used in the method above.